pandas/io/tests/test_packers.py

```diff
@@ -57,6 +59,20 @@ def check_arbitrary(a, b):
         assert(a == b)
 
 
+@contextmanager
+def patch(ob, attr, value):
+    noattr = object()  # mark that the attribute never existed
+    old = getattr(ob, attr, noattr)
+    setattr(ob, attr, value)
```

Comment:
let's put this in ``pandas.utils.testing`` (also add a doc-string), basically pointing to a compat for Mock.